### PR TITLE
Add do after to surgery

### PR DESCRIPTION
**Pull request by DrSmugleaf**
Still wireless surgery, but now with latency.

### CODE REVIEW

File: Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs [hunk 1]
```diff
@@ -1,31 +1,125 @@
...
+            {
+                // Case: skin is fully open.
+                toReturn += Loc.GetString("There is an incision on {0:their} {1}.\n", Owner, Parent.Name);
+                foreach (var mechanism in _disconnectedOrgans)
+                {
+                    toReturn += Loc.GetString("{0:their} {1} is loose.\n", Owner, mechanism.Name);
+                }
+            }
+
+            return toReturn;
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs:105]: Stringbuilder

loc at the end would be faster but this isn't called a lot so not a huge deal.

File: Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs [hunk 2]
```diff
@@ -94,94 +188,75 @@ public class BiologicalSurgeryDataComponent : SurgeryDataComponent
...
+                VesselsClamped = true;
+            }
         }
 
-        private void RetractSkinSurgery(IBodyPartContainer container, ISurgeon surgeon, IEntity performer)
+        private async void RetractSkinSurgery(IBodyPartContainer container, ISurgeon surgeon, IEntity performer)
         {
             if (Parent == null) return;
 
             performer.PopupMessage(Loc.GetString("Retract the skin..."));
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs:240]:
```suggestion
            performer.PopupMessage(Loc.GetString("Retracting the skin..."));
```

File: Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs [hunk 3]
```diff
@@ -94,94 +188,75 @@ public class BiologicalSurgeryDataComponent : SurgeryDataComponent
...
+                SkinRetracted = true;
+            }
         }
 
-        private void CauterizeIncisionSurgery(IBodyPartContainer container, ISurgeon surgeon, IEntity performer)
+        private async void CauterizeIncisionSurgery(IBodyPartContainer container, ISurgeon surgeon, IEntity performer)
         {
             if (Parent == null) return;
 
             performer.PopupMessage(Loc.GetString("Cauterize the incision..."));
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs:252]:
```suggestion
            performer.PopupMessage(Loc.GetString("Cauterizing the incision..."));
```

Ignore if surgery is getting refactored obviously.

File: Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs [hunk 4]
```diff
@@ -214,8 +289,16 @@ private void LoosenOrganSurgery(IBodyPartContainer container, ISurgeon surgeon,
 
             performer.PopupMessage(Loc.GetString("Loosen the organ..."));
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs:290]:
```suggestion
            performer.PopupMessage(Loc.GetString("Loosening the organ..."));
```

File: Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs [hunk 5]
```diff
@@ -247,20 +330,31 @@ private void RemoveOrganSurgery(IBodyPartContainer container, ISurgeon surgeon,
 
             performer.PopupMessage(Loc.GetString("Remove the organ..."));
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs:331]:
```suggestion
            performer.PopupMessage(Loc.GetString("Removing the organ..."));
```

File: Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs [hunk 6]
```diff
@@ -247,20 +330,31 @@ private void RemoveOrganSurgery(IBodyPartContainer container, ISurgeon surgeon,
...
+            }
         }
 
-        private void RemoveBodyPartSurgery(IBodyPartContainer container, ISurgeon surgeon, IEntity performer)
+        private async void RemoveBodyPartSurgery(IBodyPartContainer container, ISurgeon surgeon, IEntity performer)
         {
             if (Parent == null) return;
             if (container is not IBody body) return;
 
             performer.PopupMessage(Loc.GetString("Saw off the limb!"));
```
Comment (metalgearsloth) [inline Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs:352]:
```suggestion
            performer.PopupMessage(Loc.GetString("Sawing off the limb!"));
```

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs (hunk 1 above)
Comment (DrSmugleaf) [reply Content.Server/GameObjects/Components/Body/Surgery/BiologicalSurgeryDataComponent.cs:105]: Done.

**Comment by DrSmugleaf** [comment]
Reviews addressed.

**Review by metalgearsloth — APPROVED** [review]

**Pull request merged**